docs/src/main/asciidoc/_developer-guide/JPA_Reference_Guide.adoc

```diff
@@ -529,14 +501,18 @@ persistence property *wildfly.jpa.hibernate.search.module* to either
 *none* to not automatically inject any Hibernate Search module, or to
 any other module identifier to inject a different module. +
 For example you could set
-*wildfly.jpa.hibernate.search.module=org.hibernate.search.orm:5.4.0.Alpha1*
-to use the experimental version 5.4.0.Alpha1 instead of the provided
+*wildfly.jpa.hibernate.search.module=org.hibernate.search.orm:5.11.0.Alpha1*
+to use the experimental version 5.11.0.Alpha1 instead of the provided
 module; in this case you'll have to download and add the custom modules
 to the application server as other versions are not included. +
 When setting *wildfly.jpa.hibernate.search.module=none* you might also
 opt to include Hibernate Search and its dependencies within your
 application but we highly recommend the modules approach.
 
+[[Using latest Hibernate ORM within WildFly]]
+== Using latest Hibernate ORM within WildFly
+https://docs.jboss.org/hibernate/orm/5.3/topical/html_single/wildfly/Wildfly.html[Read about Hibernate ORM feature packs here].
```

Comment:
Let's use https://docs.jboss.org/hibernate/stable/orm/topical/html_single/wildfly/Wildfly.html to be sure it points to the latest.

Reply:
switched to https://docs.jboss.org/hibernate/stable/orm/topical/html_single/wildfly/Wildfly.html, thanks!